src/cloud/env/variables-deploy.md

```diff
@@ -194,6 +194,20 @@ stage:
       _merge: true
 ```
 
+The following example overwrites the default database connection with custom connection settings for an Amazon Aurora MySQL database:
```

Comment:
I am not sure that we already should post information about Aurora. It is planned to be supported not earlier Magento 2.4.2